app/views/encode_records/index.html.erb

```diff
@@ -23,8 +26,29 @@
 
 <% content_for :page_scripts do %>
 <script>
+
+var encodeProgressTimeout;
+
+update_encode_progress = function() {
+  let ids = $('.encode-progress').map(function(d){ return $(this).data('encode-id') }).get()
+  $.post($('#encode-records').data('progress-url'),
+    {ids},
+    function(progress_data) {
+      let stop = true;
+      $('.encode-progress').each(function(i, bar) {
+        let progress = progress_data[$(bar).data('encode-id')]
+        stop = stop && (progress == 100)
+        $(bar).css('width', progress + "%");
```

Comment:
Is this supposed to be setting the progress value instead of css width?

Reply:
Good point. I just copied what we're doing on the item view page, but it isn't using <progress>. Fixed.